nikola/plugins/command/new_post.py

```diff
@@ -213,6 +213,14 @@ class CommandNewPost(Command):
                     'bbcode, html, textile, txt2tags',
         },
         {
+            'name': 'kernel',
```

Comment:
My idea would be to make it `-f ipynb@python` — this will also be extensible in the future when we have other compilers with multiple formats (or `-f pandoc@markdown`, for example)

If we do add this parameter, it must also be added in `new_page.py`.